libs/common/src/lib/decorators/json-api-response.decorator.ts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I understand now why the rule is being disabled, Im wondering if any of those variables can be properly typed. otherwise it's ok.

Reply:
I did spend some time looking into that and didn't find a clean way. I avoid disabling this rule whenever possible.